downstream/assemblies/builder/assembly-common-ee-scenarios.adoc

```diff
@@ -0,0 +1,13 @@
+[id="assembly-common-ee-scenarios"]
+
+= Common {ExecEnvNameSing} scenarios
+
+Use the following example definition files address common configuration scenarios.
```

Comment:
I could be reading this incorrectly, but seems like there's just one word missing. If I'm incorrect, please disregard.
```suggestion
Use the following example definition files to address common configuration scenarios.
```

Reply:
Fixed.